### ISSUE TITLE
Authenticator Application continues to prompt for password after removed

### ISSUE THREAD

**Issue by wagner-robert** [Yubico/yubioath-desktop#730]
- Yubico Authenticator version: 5.0.5
- Operating system and version: Windows 10
- YubiKey model and version: v4, v5 
- Bug description summary: Application keeps prompting for password even though it was removed and reset.

Steps to reproduce
---
1) setup OATH with password
2) Use application - it will prompt for password
3) run ykman oath reset
4) run ykman and add new credentials
5) try to use Authenticator - still prompts for password

[Please explain what you did when the bug appeared, and if and how you have been
able to reproduce it.]

I have not tried to reproduce this, but notice it is prompting for password on other tokens.

Expected result
---
It should see that ykman show password as disabled and not prompt for password.

Actual results
---

[What actually happened?]

Other info
---

[Anything else you would like to add?]

**Comment by fdennis** [comment]
Are you restarting the GUI at all between step 2 and 5?

**Comment by wagner-robert** [comment]
Yes, several times.  Also tried deleting the json file in the .ykman folder.  Also restarting between keys.

RW
________________________________

**Comment by fdennis** [comment]
Have you been able to reproduce this on other computers?

**Comment by wagner-robert** [comment]
I should be able to test later today.  I do think it may be localized to one computer that I entered the password on, but will try another.

RW
________________________________

**Comment by fdennis** [comment]
Yes, please, do that and report back. I am not able to reproduce this issue.

**Comment by wagner-robert** [comment]
Can you provide details on where the secrets are stored and the logic as to whether or not to prompt for the password?
If the software is supposed to look at the key and read that Password protection = enabled/disabled then this is not occurring.
If it is looking at some registry setting : https://github.com/Yubico/yubioath-desktop/issues/390
Enhancement: Add keyring support for storing device password
then I can review those settings.

C:\Program Files\Yubico\YubiKey Manager>ykman oath info
OATH version: 4.4.5
Password protection: disabled


Robert Wagner



________________________________

**Comment by wagner-robert** [comment]
More Details... No prompt when using ykman, it does prompt when I use my own routines with yubikit library:
I am not sure why, but something I am doing forces it to prompt for password, but the yubikit library doesn't catch my error.

C:\WINDOWS\system32>ykman oath info
OATH version: 5.1.2
Password protection: disabled

                    cred_data = CredentialData(
                                oathname_v,
                                OATH_TYPE.TOTP,
                                HASH_ALGORITHM.SHA1,
                                oathsecret_v,
                                digits_v,
                                totpperiod_v,
                                counter_v,
                                issuer_v,
                            )
                
                    o = OathSession(connection)
                    o.put_credential(cred_data, touch)

cred_data	CredentialData: CredentialData(name='RBuild', oath_type=<OATH_TYPE.TOTP: 32>, hash_algorithm=<HASH_ALGORITHM.SHA1: 1>, secret=b'\\xa9\\xa1\\xa2\\x1c\\x83x\\xe4\\xff$\\xe9{Q\\xfa\\xc3\\x08\\xc4\\xb1\\x97\\x90\\x8b', digits=6, period=60, counter=0, issuer='Azure')	
	counter	int: 0	
	digits	int: 6	
	hash_algorithm	HASH_ALGORITHM: HASH_ALGORITHM.SHA1	
	issuer	str: Azure	
	name	str: RBuild	
	oath_type	OATH_TYPE: OATH_TYPE.TOTP	
	period	int: 60	
	secret	bytes: b'\\xa9\\xa1\\xa2\\x1c\\x83x\\xe4\\xff$\\xe9{Q\\xfa\\xc3\\x08\\xc4\\xb1\\x97\\x90\\x8b'

**Comment by wagner-robert** [comment]
Running with: oath accounts add -t --period 60 -i Azure RBuild VGQ2EHEDPDSP6JHJPNI7VQYIYSYZPEEL

**Comment by wagner-robert** [comment]
This is a problem with writing the same credentials over and over again, even with a ykman oath reset. (issuer and name and secret)
>ykman oath reset
---test yubi authenticator - no accounts listed
---close authenticator
>ykman oath accounts add -t --period 60 -i Azure RBuild VGQ2EHEDPDSP6JHJPNI7VQYIYSYZPEEL
---Open authenticator  - prompted for password

**Comment by fdennis** [comment]
Aha! I was now able to reproduce your issue with the command `ykman oath accounts add -t --period 60 -i Azure RBuild VGQ2EHEDPDSP6JHJPNI7VQYIYSYZPEEL`. Turns out it is the combination of the `-t` flag and a non-standard period that is giving us trouble. You probably noticed that when you started up authenticator the second time, it was waiting for something. It was actually waiting on a touch. This is not the expected behavior. We have identified the issue in ykman cli and we know what is causing it. Next release of ykman cli will have this fixed and is planned to be in Aug/Sept.
